src/mumble/EchoCancelOption.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2005-2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_ECHOCANCELLATIONOPTION_H
+#define MUMBLE_MUMBLE_ECHOCANCELLATIONOPTION_H
+
+#include <QtCore/QObject>
+
+
+/// This enum lists a series of echo cancellation options
+/// Each audio backend will determine which option is indeed available to that backend and
+/// to the specific device combination
+enum class EchoCancelOptionID { DISABLED = 0, SPEEX_MIXED = 1, SPEEX_MULTICHANNEL = 2, APPLE_AEC = 3 };
+
+struct EchoCancelOption {
+	EchoCancelOptionID id;
+	QString description;
+	QString explanation;
+};
+
+// Please strictly follow the order of the EchoCancelOptionID when adding items to this array.
+static const EchoCancelOption echoCancelOptions[] = {
+	{ EchoCancelOptionID::DISABLED,
+	  QObject::tr("Disabled"),
```

Comment:
I hope calling `tr` here is not troublesome.

Reply:
Should be fine, I think